aerotech_101smc2/aerotech_101smc2/_aerotech.py

```diff
@@ -36,21 +39,41 @@ def get_state(self):
         state["value"] = self.value
         return state
 
-
-
-
     def _set_position(self, position):
-        ...
-
-
+        self._serial_port.write(f"M {position}\n".encode())
+
+    def close(self):
+        self._serial_port.close()
+
+    def direct_serial_write(self, message):
+        self._busy = True
+        self._serial_port.write(message.encode())
+
+    def home(self):
+        # Since homing is typically a long process, start a new asynchronous task
+        # This may not be necessary, depending on how your device behaves,
+        # but remember that home is defined as returning to the current destination
+        # This method should return quickly, not wait for the homing to complete.
```

Comment:
Comments (especially the last three lines, up to you on the first) can be removed